src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = function(num = 0) {
+    sum += num;
+
+    if (num === 0) {
+      const sumCopy = sum + 0;
```

Comment:
```suggestion
      const sumCopy = sum;
```
For what you need zero?